datahub-web-react/src/app/search/utils/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { RouteComponentProps } from 'react-router';
+
+export function updateUrlParam(history: RouteComponentProps['history'], key: string, value: string) {
+    const url = new URL(window.location.href);
+    const { searchParams } = url;
+    searchParams.set(key, value);
+    history.replace(url.search);
+}
```

Comment:
This feels even more generic than search and could go in a higher-level utils file somewhere.

Reply:
I'm curious how the browse sidebar behaves when we mutate url params without adjusting a filter.... should be fine but I know we got a useEffect in there that kicks off some stuff when params change.